packages/form-js-viewer/src/render/hooks/useCleanupMultiSelectValues.js

```diff
@@ -9,10 +10,12 @@ export default function(props) {
     options,
     loadState,
     onChange,
-    values
+    values: valuesArray
```

Comment:
I just need an intermediate because we're redefining values lower down. Wdyt we should call it? Or should we rename the new values to something else instead?

Reply:
I've renamed the other values to memoizedValues